File: engine/src/main/java/com/terralite/engine/input/InputActions.java
package com.terralite.engine.input;

public final class InputActions {
    public static final InputAction MOVE_FORWARD = InputAction.of("terralite:move_forward");
    public static final InputAction MOVE_BACK = InputAction.of("terralite:move_back");
    public static final InputAction MOVE_LEFT = InputAction.of("terralite:move_left");
    public static final InputAction MOVE_RIGHT = InputAction.of("terralite:move_right");
    public static final InputAction JUMP = InputAction.of("terralite:jump");
    public static final InputAction SPRINT = InputAction.of("terralite:sprint");
    public static final InputAction BREAK_BLOCK = InputAction.of("terralite:break_block");
    public static final InputAction PLACE_BLOCK = InputAction.of("terralite:place_block");
    public static final InputAction TOGGLE_CAMERA = InputAction.of("terralite:toggle_camera");

    private InputActions() {
    }
}

File: engine/src/commonMain/kotlin/com/terralite/engine/input/InputActions.kt
package com.terralite.engine.input

object InputActions {
    @JvmField val MOVE_FORWARD: InputAction = InputAction.of("terralite:move_forward")
    @JvmField val MOVE_BACK: InputAction = InputAction.of("terralite:move_back")
    @JvmField val MOVE_LEFT: InputAction = InputAction.of("terralite:move_left")
    @JvmField val MOVE_RIGHT: InputAction = InputAction.of("terralite:move_right")
    @JvmField val JUMP: InputAction = InputAction.of("terralite:jump")
    @JvmField val SPRINT: InputAction = InputAction.of("terralite:sprint")
    @JvmField val BREAK_BLOCK: InputAction = InputAction.of("terralite:break_block")
    @JvmField val PLACE_BLOCK: InputAction = InputAction.of("terralite:place_block")
    @JvmField val TOGGLE_CAMERA: InputAction = InputAction.of("terralite:toggle_camera")
}
